meinberlin/apps/votes/models.py

```diff
@@ -29,23 +32,25 @@ class VotingToken(models.Model):
         ),
     )
 
-    class Meta:
-        unique_together = ["token", "module"]
-
     # if token already exists, try to generate another one
+    # FIXME: this is never called during generation, remove?
     def save(self, *args, **kwargs):
-        for i in range(4):
-            try:
-                super().save(*args, **kwargs)
-                return
-            except IntegrityError as e:
-                if "UNIQUE constraint failed" in e.args[0]:
-                    if i == 3:
-                        raise
-                    else:
-                        self.token = get_token()
-                else:
-                    raise
+        # if self._state == 'adding':
```

Comment:
do we still need this? codes are only generated using the batch_create, which doesn't call save(), right?